README.md

```diff
@@ -10,36 +10,36 @@ different criteria such as scope, author, year, etc.
 Requirements
 ------------
 
-* Django 1.10 - 2.0
-* Python 2.7, 3.4 - 3.6
+* Django 1.11, 2.0
+* Python 2.7, 3.5- 3.7
 
 
 Installation
 ------------
 
 1.  Download and install from source code.
     ```sh
-	$ pip install git+git://github.com/unt-libraries/django-citeit.git
+	$  pip install git+git://github.com/unt-libraries/django-citeit.git
```

Comment:
I think this extra space isn't needed.

Reply:
Well, I added it 'cos the first letters were missing in the rendered part of README in my IDE. And adding it didn't change the rendering in GitLab, so I thought adding it wouldn't hurt.